docs/book/src/clusterctl/developers.md

```diff
@@ -139,25 +139,29 @@ See [Install and/or configure a kubernetes cluster] for more about how.
 Before running clusterctl init, you must ensure all the required images are available in the kind cluster.
 
 This is always the case for images published in some image repository like docker hub or gcr.io, but it can't be
-the case for images built locally; in this case, you can use `kind load` to move the images built locally. e.g 
- 
+the case for images built locally; in this case, you can use `kind load` to move the images built locally. e.g
+
 ```
 kind load docker-image gcr.io/k8s-staging-cluster-api/cluster-api-controller-amd64:dev
 kind load docker-image gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller-amd64:dev
 kind load docker-image gcr.io/k8s-staging-cluster-api/kubeadm-control-plane-controller-amd64:dev
 kind load docker-image gcr.io/k8s-staging-cluster-api/capd-manager-amd64:dev
-``` 
+```
 
 to make the controller images available for the kubelet in the management cluster.
 
 ## Additional Notes for the Docker Provider
 
+### Note 1
```

Comment:
What do you think about having these headings as titles instead of 1/2/3?

Reply:
Sure! 👍